hrm-domain/hrm-service/src/contact/contact.ts

```diff
@@ -267,6 +268,18 @@ export const createContact = async (
       conversationMediaPayload,
     } = getNewContactPayload(newContact);
 
+    const profileResult = await getOrCreateProfileWithIdentifier(conn)(
```

Comment:
Absolutely fine for now but I wonder in the future if profile creation should be tied to contact creation? We've separated transcript creation now, and in the future profiles might be auto-generated on the back end?

There is also the case of offline contacts, presumably those will have their identifier added manually by the counsellor after they are created, so will need to be linked up after the fact.

I know this is all out of scope for this PR and this totally makes sense for now, just thinking aloud

Reply:
The initial work is basically to get to feature parity with the current "previous contacts" banner, but using robust backend systems that will allow for expansion. Profile creation will definitely not be tied one to one like this with contact creation in the future... exactly what it will look like is a bit up in the air still. ;)